.circleci/config.yml

```diff
@@ -6,27 +6,35 @@ orbs:
 workflows:
   catalog_build:
     jobs:
-      - orb-tools/lint
+      - orb-tools/lint:
+          filters:
+            tags:
+              only: /.*/
 
       - orb-tools/pack:
           source-dir: src
           destination-orb-path: packed/orb.yml
           workspace-path: packed/orb.yml
           artifact-path: packed/orb.yml
           requires: [orb-tools/lint]
+          filters:
+            tags:
+              only: /.*/
 
       - orb-tools/publish-dev:
           orb-name: giantswarm/architect
           orb-path: workspace/packed/orb.yml
           publish-token-variable: CIRCLECI_DEV_API_TOKEN
           requires: [orb-tools/pack]
 
-      - orb-tools/dev-promote-prod:
-          orb-name: giantswarm/architect
+      - orb-tools/publish:
+          attach-workspace: true
+          orb-path: workspace/packed/orb.yml
+          orb-ref: giantswarm/architect@${CIRCLE_TAG##v}
           publish-token-variable: CIRCLECI_DEV_API_TOKEN
-          release: patch
-          requires: [orb-tools/publish-dev]
+          requires: [orb-tools/pack]
           filters:
             branches:
-              #only: master
               ignore: /.*/
+            tags:
+              only: /.*/
```

Comment:
For this job I think we should be extra safe use some better filtering for versions. Like `/^v\d+\.\d+\.\d+$/`.

Reply:
Yeah good point.

Just for info circleci also validates the tag format:
```
Error: Invalid version specified for publishing: my-test (expected either a semver-style version string in x.y.z format, or a development-style version string in dev:* format.)
```